weg-li/Features/PersonalData/PersonalDataViewModel.swift

```diff
@@ -71,12 +68,6 @@ final class PersonalDataViewModel: ObservableObject {
             .assign(to: \.isPhoneValid, on: self)
             .store(in: &bag)
         
-        $email
-            .removeDuplicates()
```

Comment:
wieso wird denn hier über so nen kram iteriert? sind das arrays?

Reply:
Das hier ist an sich nicht die E-Mail Adresse in `$email`, da steckt durch die Struktur das hier drin:

```
Publisher(subject: Combine.CurrentValueSubject<Swift.String, Swift.Never>)
```

Reply:
😕

Reply:
https://developer.apple.com/documentation/combine/currentvaluesubject/removeduplicates()